instances/latest/terminologies/technique/darkFieldMicroscopy.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/darkFieldMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
```

Comment:
Description: Darkfield microscopy is an optical microscopy techniques that generates contrast by differentially filtering scatter and unscattered light. Specifically it transmits scattered light and blocks unscattered light. The effect is to make the areas of a transparent sample that scatter light appear brighter than those that do not scatter light. A dark background is used (i.e. the light source is not directly behind the sample in the optical path) so that unscattered light does not overwhelm the scattered light.

Reply:
This is a subClassOf light microscopy technique.

Reply:
Darkfield microscopy is an optical microscopy technique in which illumination light is transmitted through the sample so that it does not directly enter the optics and contrast is generated by the differential scattering of light within the specimen.

Brightfield microscopy is an optical microscopy technique in which illumination light is transmitted through the sample so that it directly enters the optics and contrast is generated by the differential absorption of light within the specimen.